sdk/servicebus/service-bus/package.json

```diff
@@ -87,7 +87,7 @@
     "dist-esm/package.json",
     "dist-esm/src/",
     "src/",
-    "typings/src",
+    "typings",
```

Comment:
Why this change? This will include the typings for test files as well

Reply:
API extractor .d.ts file was going into "typings" (which will be renamed to "types"), leaving "typings/src" to be all the .d.ts files created by the compiler. 

We may want to put the .d.ts files for tests elsewhere, if it's not something we're shipping.

Reply:
> We may want to put the .d.ts files for tests elsewhere, if it's not something we're shipping

We do ship the d.ts files

> API extractor .d.ts file was going into "typings" (which will be renamed to "types"), leaving "typings/src" to be all the .d.ts files created by the compiler.

In that case lets revert this back to `typings/src` to avoid test d.ts files being shipped and add another entry for `typings/the-d.ts-file-that-is-generated`

Reply:
I see that `typings/src` is back. Thats great

Since we have changed the `types` field in these 3 libraries (event hubs, EPH and Service Bus) to point to the d.ts file generated by the extractor, we should ensure that they are included in the `files` property in  package.json. If we are not ready to ship the file generated by the extractor, then we shouldnt change the `types` field either